package.json

```diff
@@ -27,39 +27,40 @@
     "lwm2mAgent": "bin/lwm2mAgent.js"
   },
   "engines": {
-    "node": ">=0.10.8"
+    "node": ">=4.8.4"
```

Comment:
I think this has been just merged as part of PR #92... maybe this PR's branch should be upgrades with master so this change would not be shown?

Reply:
Fixed in c86cf5c10739653161112be9133ff978bb8c386c